### ISSUE TITLE
Any way to increase font size?

### ISSUE THREAD

**Issue by funkenstrahlen** [remuslazar/osx-xliff-tool#21]
Is there any way to change the font size? On a large display it is kinda small.

**Comment by remuslazar** [comment]
good point, I will implement it using the well-known "browser" shortcuts for that (Cmd +/-, Cmd 0). Thanks!

**Comment by funkenstrahlen** [comment]
Using shortcuts is great! Maybe it is also helpful to have this options visible in the menubar to discover them more easily.

**Comment by remuslazar** [comment]
sure thing, yes.
